plugins/source/github/client/client.go

```diff
@@ -74,6 +90,28 @@ func Configure(ctx context.Context, logger zerolog.Logger, s specs.Source, _ sou
 			Repositories:  c.Repositories,
 			Teams:         c.Teams,
 		},
-		Orgs: spec.Orgs,
+		orgs:            spec.Orgs,
+		orgRepositories: orgRepositories,
 	}, nil
 }
+
+func discoverRepositories(ctx context.Context, client *github.Client, orgs []string) (map[string][]*github.Repository, error) {
+	opts := &github.RepositoryListByOrgOptions{ListOptions: github.ListOptions{PerPage: 100}}
+
+	orgRepos := make(map[string][]*github.Repository)
+	for _, org := range orgs {
+		for {
+			repos, resp, err := client.Repositories.ListByOrg(ctx, org, opts)
```

Comment:
Since we're here.... it would be really nice if this also supported personal accounts, not just orgs 😄 

But happy to leave that for a future update also

Reply:
Yeah, I think it's another feature, as our spec says `orgs` and we'll need to verify the API calls in all the resources (I think not all GitHub APIs are supported for personal accounts)